wpeverest/everest-forms
# Enhancement - Date time slot booking

### All Submissions:


### Changes proposed in this Pull Request:
This PR adds the new slot booking feature in date/time field.

### How to test the changes in this Pull Request:

1. Edit or create a form then drag the date/time field.
2. Goto the field option -> check the slot booking option in the advanced setting.
3. Submit the form by choosing a date. and again try to submit by choosing the exact date. verify the message and repeat the process for different formats and modes of date/time.

### Types of changes:


### Other information:


### Changelog entry

>  Enhancement - Date time slot booking
